components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/WebsocketHandler.java

```diff
@@ -49,6 +49,14 @@ public WebsocketHandler() {
         }
```

Comment:
Shall we call the other constructor from this constructor to avoid code duplication?

Reply:
Sure. Fixed via https://github.com/wso2/carbon-apimgt/pull/11083/commits/f6e4756e3c4828cc8adae266cee7af6384dd957e